index.js

```diff
@@ -47,9 +47,14 @@ db.query("SELECT * FROM student;", (error, results, fields) => {
     if (error) throw error
     console.log(results)
 })
+db_stu.query("SELECT * FROM student;", (error, results, fields) => {
+    if (error) throw error
+    console.log(results)
+})
 
 // Require routes
-require("./src/routes")(app, db)
+require("./src/Routers/routes")(app, db)
+require("./src/Routers/student_getall")(app, db_stu)
```

Comment:
Better to name as `routes/routes_admin` .
we surely wont be making different file for different endpoints.